src/components/navigation.tsx

```diff
@@ -9,27 +9,44 @@ type Props = {
 };
 
 const Navigation = ({ sections, currentSlug }: Props) => {
-  const [isOpen, setIsOpen] = useState(false);
+  const [isOpen, setIsOpen] = useState<boolean>(false);
   const toggle = () => setIsOpen(!isOpen);
   const onItemClick = () => {
     if (isSmallScreen()) {
       toggle();
     }
   };
   const className = isOpen ? 'side-nav side-nav--open' : 'side-nav';
+  let flatSections: NavigationSectionItem[] = [];
+  Object.keys(sections).map((sectionKey: string) => {
```

Comment:
nit: Incorrect usage of `map` (return value missing). Change to the more appropriate `forEach` instead.

```suggestion
  Object.keys(sections).forEach((sectionKey: string) => {
```

Reply:
Looks like this got undone with the force push.